.github/sync-files.yaml

```diff
@@ -11,10 +11,12 @@
     - source: .github/PULL_REQUEST_TEMPLATE/small-change.md
     - source: .github/PULL_REQUEST_TEMPLATE/standard-change.md
     - source: .github/dependabot.yaml
+    - source: .github/stale.yml
     - source: .github/workflows/pre-commit.yaml
     - source: .github/workflows/pre-commit-optional.yaml
     - source: .github/workflows/semantic-pull-request.yaml
     - source: .github/workflows/spell-check-differential.yaml
+    - source: .github/workflows/sync-files.yaml
```

Comment:
Hmm this is the first time we are adding `sync-files.yaml` to this repository.

Reply:
Yes, this repository was the sync source before https://github.com/autowarefoundation/autoware/pull/110.

Reply:
Thanks for clarifying!